core/src/replay_stage.rs

```diff
@@ -2770,21 +2770,26 @@ impl ReplayStage {
     ) -> GenerateVoteTxResult {
         if !bank.has_initial_accounts_hash_verification_completed() {
             info!("startup verification incomplete, so unable to vote");
-            return GenerateVoteTxResult::Failed;
+            return GenerateVoteTxResult::Failed("startup verification incomplete".into());
         }
 
         if authorized_voter_keypairs.is_empty() {
             return GenerateVoteTxResult::NonVoting;
         }
         if let Some(slot) = wait_to_vote_slot {
             if bank.slot() < slot {
-                return GenerateVoteTxResult::Failed;
+                return GenerateVoteTxResult::Failed(format!(
```

Comment:
should this be `GenerateVoteTxResult::WaitToVoteSlot`?

Reply:
Oops, good catch, changed.